homeassistant/components/elmax/config_flow.py

```diff
@@ -151,7 +151,9 @@ async def _test_direct_and_create_entry(self):
                     port=self._panel_direct_port,
                 )
             )
-            ssl_context = build_direct_ssl_context(cadata=self._panel_direct_ssl_cert)
+            ssl_context = await async_build_direct_ssl_context(
```

Comment:
Is there a reason why we simply aren't calling the original `build_direct_ssl_context` using `hass.async_add_executor_job`?

e.g.:

```python
ssl_context = await self.hass.async_add_executor_job(build_direct_ssl_context, self._panel_direct_ssl_cert)
```

?

Reply:
Good idea. Switched to execturo_job in e23d5bb. Thanks @frenck !